dist/amd64/birdy/bird-wrapper.sh

```diff
@@ -1,8 +1,18 @@
 #!/bin/sh
 
-set -e
+set -ex
+
+ROUTER_ID=`ip -4 -o a | while read num intf inet addr; do
```

Comment:
Please can you expand commands like `ip a` to make them easier for newbies to understand?  It's easier to guess what `ip addr` does than `ip a` and maybe give sample output in a comment to make it easier to see what we're matching against below.